models/classes/LtiLaunchData.php

```diff
@@ -65,6 +65,9 @@ class LtiLaunchData implements \JsonSerializable
     const LTI_SHOW_SCORE = 'custom_show_score';
     const LTI_SHOW_CORRECT = 'custom_show_correct';
 
+    // AGS
+    const AGS_CLAIMS = 'ags_claims';
```

Comment:
I would add something like
```suggestion
    public const AGS_CLAIMS = 'ags_claims';
```